data/mods/MindOverMatter/monsters/monster_overrides.json

```diff
@@ -308,11 +308,18 @@
     "extend": { "flags": [ "TEEP_IMMUNE", "TELEPORT_IMMUNE" ], "armor": { "psi_telekinetic_damage": 50 } },
     "attack_effs": [ { "id": "effect_teleporting_lock", "duration": [ 5, 60 ], "chance": 50 } ]
   },
+  {
+    "id": "mon_tindalos",
+    "copy-from": "mon_tindalos",
+    "type": "MONSTER",
+    "extend": { "flags": [ "TEEP_IMMUNE", "TELEPORT_IMMUNE" ], "armor": { "psi_telekinetic_damage": 50 } },
+    "attack_effs": [ { "id": "effect_teleporting_lock", "duration": [ 5, 60 ], "chance": 50 } ]
+  },
```

Comment:
Was it your intention to make two different definition for mon_tindalos?
![image](https://github.com/user-attachments/assets/570a99fd-15d4-4c52-9234-4277341ac4a1)